slider/src/components/SliderPlugin.vue

```diff
@@ -16,44 +29,57 @@ export default {
       default: undefined,
     },
   },
+
   computed: {
+    minValue() {
+      return this.getMinValue()
```

Comment:
This can be rewritten like this:

```suggestion
      return this.options.minValue
        ? Number.parseFloat(this.options.minValue)
        : 0
```

And then `getMinValue()` can be removed.

The same applies for `getMaxValue` and `getMarks`.